### ISSUE TITLE
Bring back scale?

### ISSUE THREAD

**Issue by Codezilluh** [Prozi/detect-collisions#32]
I know this was originally a fork of [Sinova/Collisions](https://github.com/Sinova/Collisions) (I used this a few years ago and liked it) and it appears you have made a lot of improvements since the last time I used this; however, it also appears that `body.scale`, `body.scale_x`, and `body.scale_y` are now gone. And now I am just wondering if it is possible to bring them back or not. Were they removed for a technical reason? Thanks.

**Comment by Prozi** [comment]
did you consider scale useful for bodies?

I can think of writing it back it should be fairly simple.

the api will slightly change, propably, like no snake_case in param names now

it was removed because when I moved to around I think v3.0 of this library I decided to use 2 ready to use and popular, maintained libraries, and combine them to do the same. those were rbush for BVH and sat for SAT. some features differ between that version and current one. but the current one is propably more optimized and for sure feature rich (like angles, ellipses, raycasting)
